pkgs/tools/misc/watchexec/default.nix

```diff
@@ -1,23 +1,30 @@
-{ stdenv, rustPlatform, fetchFromGitHub }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, CoreServices, CoreFoundation }:
 
-rustPlatform.buildRustPackage rec {
+with rustPlatform;
+
+buildRustPackage rec {
   name = "watchexec-${version}";
-  version = "1.9.0";
+  version = "1.8.6";
+  buildInputs = stdenv.lib.optional stdenv.isDarwin [ CoreServices ];
+
+  preConfigure = ''
+    export NIX_LDFLAGS="-F${CoreFoundation}/Library/Frameworks -framework CoreFoundation $NIX_LDFLAGS"
+  '';
 
   src = fetchFromGitHub {
-    owner = "watchexec";
+    owner = "mattgreen";
     repo = "watchexec";
-    rev = version;
-    sha256 = "0zp5s2dy5zbar0virvy1izjpvvgwbz7rvjmcy6bph6rb5c4bhm70";
+    rev = "${version}";
+    sha256 = "1jib51dbr6s1iq21inm2xfsjnz1730nyd3af1x977iqivmwdisax";
   };
 
-  cargoSha256 = "1li84kq9myaw0zwx69y72f3lx01s7i9p8yays4rwvl1ymr614y1l";
+  cargoSha256 = "0sm1jvx1y18h7y66ilphsqmkbdxc76xly8y7kxmqwdi4lw54i9vl";
 
   meta = with stdenv.lib; {
     description = "Executes commands in response to file modifications";
-    homepage = https://github.com/watchexec/watchexec;
+    homepage = https://github.com/mattgreen/watchexec;
```

Comment:
This should not be changed since github redirects mattgreen/watchexec to watchexec/watchexec.